domain/machine/state/state.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	coredb "github.com/juju/juju/core/database"
 	"github.com/juju/juju/core/logger"
+	coremachine "github.com/juju/juju/core/machine"
```

Comment:
I don't think you need to alias this everywhere when there is no conflict with another package import.